presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1189,6 +1189,12 @@ public void testPartitionPerScanLimit(Session session, HiveStorageFormat storage
             assertUpdate(session, insertPartitions, 100);
         }
 
+        // verify can show partitions
```

Comment:
add a test `SHOW PARTITIONS FROM table-name` for a table with partition count over limit.
IMO it should succeed. My reasoning is that `hive.max-partitions-per-scan` (as the name suggests) limits number of partitions (the data) read by a scan (query). Here, we are not reading partitions (the data) but only partition names, so the limit should not apply.